triveni-codev/GBWebApp-dev
# Darlp networks service extension 1203841430074422

Hi @triveni-codev ,

Kindly have someone to check this PR for ATSC3 Service Extension Types: Broadband Signaling, Broadband Service Icon and Base Pattern.

ASANA: https://app.asana.com/0/1201943683850147/1203841430074422


PR Considerations:
-this is limited only for atsc 3 network type
-this is limited only for Broadband Signaling, Broadband Service Icon and Base Pattern Extension types

To Test:
-adding and updating of these mentioned extension types on DIRTY change (uncommitted)
-can retrieve any of these 3 existing service extension types from server, display and can be updated
-any of the extension types can have one or more instance (you can add one or more)
-saving/updating and cancelling should be good on child modal until the topmost parent modal.

DONT Test big commit for adding and updating yet. Still in development.
